src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -107,7 +109,14 @@
         public final ICubeRenderer solidCasingRenderer;
         public final SimpleCubeRenderer fireboxIdleRenderer;
         public final SimpleCubeRenderer firefoxActiveRenderer;
-
+        //Left not private to add default for compatibility with addons
+        public OrientedOverlayRenderer frontOverlay = Textures.MULTIBLOCK_WORKABLE_OVERLAY;
```

Comment:
I had to leave this as not final to have a default, so that it will default to using this front overlay for addons that may add a new boiler while using the old constructor, but still be able to assign the passed overlay to it in the new constructor.

One thing that I can do is make it private though, since the correct way to set the texture would be through the new constructor.

Reply:
You can make it final and still have default value if it is set in constructor, which in this case will be in deprecated one.
Let me add it via code suggestions.